core/src/edu/asu/c3simulator/simulation/Employee.java

```diff
@@ -0,0 +1,248 @@
+package edu.asu.c3simulator.simulation;
+
+import java.util.Random;
+
+public class Employee
+{
+	public String employeeName;
+	public String employeePosition;
+	public String randomEmployeePosition;
+	public int employeePay;
+	public int employeePref;
+	public int employeeMorale;
+	public int netSalary;
+	public float averageAnnualRaise;
+	public float averageAnnualBonus;
+	public int netBonuses;
+	
+	public Employee(String name, String position, int pay, int pref, int morale,
+			int net_salary, float annualRaise, float annualBonus, int net_bonus)
+	{
+		employeeName = name;
+		employeePosition = position;
+		employeePay = pay;
+		employeePref = pref;
+		employeeMorale = morale;
+		netSalary = net_salary;
+		averageAnnualRaise = annualRaise;
+		averageAnnualBonus = annualBonus;
+		netBonuses = net_bonus;
+	}
+	
+	@Override
+	public String toString()
+	{
+		StringBuilder string = new StringBuilder();
+		string.append(employeeName);
+		string.append(" ($");
+		string.append(employeePay);
+		string.append(" / hr)");
+		
+		return string.toString();
+	}
+	
+	public String getEmployeeName()
+	{
+		return employeeName;
+	}
+	
+	public void setEmployeeName(String employeeName)
+	{
+		this.employeeName = employeeName;
+	}
+	
+	public static String getRandomEmployeeName()
+	{
+		String[] firstName = { "Nicholas", "Colton", "Sawyer", "Frankie", "Jennifer",
+				"Charles", "Kathy", "James", "Jacob", "Crystal", "Mike", "Cody", "Jorge",
+				"Sam", "Lisa", "Margaret", "Marth", "Roy", "Victoria", "Susan", "Ted" };
+		String[] lastName = { "Krogstad", "Tucker", "Hardenbech", "Connelley", "Li",
+				"Baker", "Tyke", "Mattingly", "Feng", "Lee", "Smith", "Houston",
+				"Franco", "White", "Andrade", "Manning", "Brady", "Boyle", "Terry" };
+		
+		int randomFirst = (int) (Math.random() * firstName.length);
+		int randomLast = (int) (Math.random() * lastName.length);
+		
+		String randomEmployeeName = firstName[randomFirst] + " " + lastName[randomLast];
+		
+		return randomEmployeeName;
+	}
+	
+	public void setRandomEmployeeName()
```

Comment:
Replace this with a static method called generateRandomEmplyoee(), and have it create a new employee, rather than just changing the name of an already instantiated object